### ISSUE TITLE
Increase limit of videos to 100

### ISSUE THREAD

**Issue by SimeonGriggs** [mittnavnermike/gatsby-plugins#10]
Great plugin, totally works.

By default however the API only returns 25 videos when it can do up to 100. I modified the plugin to do this below and it totally worked to pull in 100.

```
client.request({  
  method: 'GET',
  path: '/me/videos?per_page=100'
},
```

The API also supports pagination but that would take more work to implement in the plugin, I imagine.

**Comment by mittnavnermike** [comment]
Cool, nice addition. I'm currently not using gatsby anymore, but PR's are very welcome!

**Comment by SimeonGriggs** [comment]
Don't think I ended up using this plugin on the site I was working on but maybe this'll help someone else in future

See: https://github.com/mittnavnermike/gatsby-plugins/pull/15

**Comment by mittnavnermike** [comment]
Merged #15
